crates/protocol/src/channel_out.rs

```diff
@@ -187,6 +188,26 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_channel_out_reset() {
+        let config = RollupConfig::default();
+        let mut channel = ChannelOut {
+            id: ChannelId::default(),
+            config: &config,
+            rlp_length: 10,
+            closed: true,
+            frame_number: 11,
+            compressor: MockCompressor::default(),
+        };
+        channel.reset();
+        assert_eq!(channel.rlp_length, 0);
+        assert_eq!(channel.frame_number, 0);
+        // The odds of this flaking are so astronomically low
```

Comment:
If you could replace "this flaking" with smthg like "the rhs and lhs being equal" the comment would be much more helpful